lib/features/pokemon_overview/pokemon_overview_page.dart

```diff
@@ -7,5 +7,72 @@ class PokemonOverviewPage extends StatelessWidget {
   });
 
   @override
-  Widget build(BuildContext context) => const Center(child: Text(str.pokedexTitle));
+  Widget build(BuildContext context) {
+    return MaterialApp(
+      home: Scaffold(
+        appBar: AppBar(
+          title: const Text(str.pokedexTitle),
+        ),
+        body: GridView.count(
+          crossAxisCount: 2,
+          children: List.generate(20, (index) {
+            return GestureDetector(
+              child: Container(
```

Comment:
You can add the onTap parameter for this but you need to add //TODO for the action